manifests/b/BizagiLimited/BizagiModeler/3.9.0015/BizagiLimited.BizagiModeler.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BizagiLimited.BizagiModeler
+PackageVersion: 3.9.0015
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://glo0modelerstordown0prod.blob.core.windows.net/cont-prod/BizagiModelerSetup_3.9.0015.exe?sv=2018-03-28&sr=b&sig=x1lT4MhLKvzdTlrB7AZM9tZUkzQLQl7Wx3qirpMl7z8%3D&st=2022-04-25T13%3A55%3A35Z&se=2022-04-25T14%3A55%3A35Z&sp=r&ext=.exe

```

Comment:
Installer URL contains a signature which becomes invalid after some time; A persistent link must be used